packages/ui/src/components/va-button/VaButton.vue

```diff
@@ -431,8 +431,8 @@ export default defineComponent({
     }
 
     &.va-button--icon-only {
-      min-width: var(--va-button-lg-size);
-      min-height: var(--va-button-lg-size);
```

Comment:
Min-* were here to prevent ellipsis bug in some cases.

Reply:
I think it won't happen, because when we set the size for the button be also setting size to the icon.